core/protobuf/src/proto/dataserver/protocol.proto

```diff
@@ -5,53 +5,62 @@ package alluxio.proto.dataserver;
 import "status.proto";
 
 // The read/write request type. It can either be an Alluxio block operation or a UFS file operation.
-// next available id: 3
+// next available id: 2
 enum RequestType {
   ALLUXIO_BLOCK = 0;
   UFS_FILE = 1;
-  UFS_BLOCK = 2;
 }
 
 // The read request.
-// next available id: 9
+// next available id: 7
 message ReadRequest {
-  optional RequestType type = 1;
```

Comment:
I deleted these deprecated fields directly and updated tags because the data server won't be backward compatible anyway in 1.5.0.